src/tests/work-item/work-item-list/quickadd-workitem.spec.js

```diff
@@ -39,18 +38,17 @@ describe('Work item list', function () {
   });
 
   it('Creating a new quick add work item and Cancel delete - phone.', function () {
-    setBrowserMode('phone');
```

Comment:
Yeah, so let's say the default size is phone. So, 
It was doing - 
```
setBrowserMode('phone'); // from beforeEach
setBrowserMode('phone');
```
Does it make sense?